src/news/ftb-removal.md

```diff
@@ -0,0 +1,45 @@
+---
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]
+
+Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs directly will no longer be possible.
+
+We will start working on a feature that allows importing FTB Modpacks from the FTB App, as this is acceptable to the FTB team.
+
+The reason for this request from the FTB team seems to be related to some launchers attempting to monetize FTB Packs.
+
+## Conversations With FTB Team
+
+In the pictures some parts are blurred out due to Privacy concerns.
+
+### Initial Permission Request
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request1.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request2.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request3.png" style="width: 40em">
+
+### Request to Remove FTB Modpack Downloading
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request4.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request5.png" style="width: 40em">
+
+### Additional
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">
```

Comment:
The alt tags should probably be more descriptive, so people with vision impairments can let the screen reader read out the screenshots.